eslint.config.mjs

```
@@ -16,7 +16,7 @@ export default [
   sonarjs.configs.recommended,
   ...tseslint.configs.recommended,
   {
-    files: ['**/utils/markdown.ts'],
+    files: ['**/utils/html_render.ts'],
```

ESLint

[error] Replace `'**/utils/html_render.ts'` with `"**/utils/html_render.ts"`

(prettier/prettier)